c/httpfileservice.c

```diff
@@ -452,6 +460,76 @@ void respondWithUnixFileMetadata(HttpResponse *response, char *absolutePath) {
   }
 }
 
+void directoryChangeOwnerAndRespond(HttpResponse *response, char *path,
+        char *usrId, char *grpId, char *Recursive, char *pattern) {
```

Comment:
What is `usrId` used for? Are we mixing UID and TSO user name?

Reply:
The usId and the grpId follows the man pages for chown, where each could be the name of the user/group or their numeric value.